app/src/main/res/values-pt/strings-pt.xml

```diff
@@ -1,496 +1,521 @@
-	<string name="grid_glucose_lines_visible">Linhas de glicose grade visível</string>
-	<string name="useful_for_noise_and_missed_readings">Útil para ruído e leituras perdidas</string>
-	<string name="the_standard_xdrip_calculated_value">Os valores padrão calculados xDrip</string>
-	<string name="display_ideal_glucose_target_line">Monitor da linha de meta ideal de glicose</string>
-	<string name="display_eight_hour_average_line">Exibição de linha média 8 horas</string>
-	<string name="display_twenty_four_hour_average_line">Exibição de linha média 24 horas</string>
-	<string name="predictive_model_inner_workings">Modelo preditivo funcionamento interno</string>
-	<string name="noise_model_inner_workings">Modelo de ruído funcionamento interno</string>
-	<string name="glucose_number_from_filtered">Número de glicose a partir de filtrado</string>
-	<string name="if_the_phone_has_text_to_speech">Se o telefone tem recursos de text-to-speech ele vai falar cada nova leitura.</string>
-	<string name="upload_logs">Carregar Logs</string>
-	<string name="short_off_text_for_switches">FORA</string>
-	<string name="short_on_text_for_switches">EM</string>
-	<string name="install_pebble_apps">Instale Pebble Apps</string>
-	<string name="sliding_24_hour_window_summary">Use últimas 24 horas em vez de tempo desde a meia-noite para as estatísticas.</string>
-	<string name="sliding_24_hour_window">Janela de correr</string>
-	<string name="split">Dividido</string>
-	<string name="delete">Excluir</string>
-	<string name="title_activity_btglucose_meter">Medidor de digitalização</string>
+
+    <string name="title_activity_bluetooth_scan">Procura Bluetooth</string>
+    <string name="connected">Conectado</string>
+    <string name="disconnected">Desligado</string>
+    <string name="menu_scan">Procurar</string>
+    <string name="menu_stop">Parar</string>
+    <string name="menu_refresh">Atualizar</string>
+    <string name="menu_export_database">Exportar base de dados</string>
+    <string name="menu_export_csv_sidiary">Exportar CSV (formato SiDiary)</string>
+    <string name="menu_import_db">Importar base de dados</string>
+    <string name="error_bluetooth_not_supported">Erro: Bluetooth não é suportado por este dispositivo</string>
+    <string name="unknown_device">Dispositivo não reconhecido</string>
+    <string name="connecting_to_device">Conectando-se ao dispositivo</string>
+    <string name="title_activity_raw_data_table">Tabela de dados brutos</string>
+    <string name="title_activity_calculated_data_table">Tabela de dados calculados</string>
+    <string name="title_activity_start_new_sensor">Iniciar Novo Sensor</string>
+    <string name="title_activity_new_sensor_location">Novo local de Sensor</string>
+    <string name="title_activity_stop_sensor">Parar Sensor</string>
+    <string name="title_activity_nav_drawer">Nav Drawer</string>
+    <string name="navigation_drawer_open">Abrir Nav Drawer</string>
```

Comment:
Can we have the full word Navigation instead of an abbreviation like the english version?